docs/en/integrations/data-visualization/grafana/config.md

```diff
@@ -0,0 +1,234 @@
+---
+sidebar_label: Plugin Configuration
+sidebar_position: 3
+slug: /en/integrations/grafana/config
+description: Configuration options for the ClickHouse data source plugin in Grafana
+---
+import ConnectionDetails from '@site/docs/en/_snippets/_gather_your_details_native.md';
+
+# Configuring ClickHouse data source in Grafana
+
+The easiest way to modify a configuration is in the Grafana UI on the plugin configuration page, but data sources can also be [provisioned with a YAML file](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources).
+
+This page shows a list of options available for configuration in the ClickHouse plugin, as well as config snippets for those provisioning a data source with YAML.
+
+For a quick overview of all the options, a full example config can be found [here](#full-example-config).
+
+## Common Settings
+
+Example configuration screen:
+<img src={require('./images/config_common.png').default} class="image" alt="Example secure native config" />
+
+Example configuration YAML for common settings:
+```yaml
+jsonData:
+  host: 127.0.0.1 # (required) server address.
```

Comment:
since you provided an example for a Cloud instance on the `Getting started` page, let's do it here as well